LEAF_Request_Portal/index.php

```diff
@@ -523,6 +523,7 @@ function customTemplate($tpl)
 $main->assign('menu', $o_menu);
 $main->assign('tabText', XSSHelpers::sanitizeHTML($tabText));
 
+$main->assign('leafSecure', Config::$leafSecure);
```

Comment:
There should be a check here to see if that `$leafSecure` variable is set. If not, it should default to `false`. This will prevent errors in case any configs aren't updated properly.

Reply:
done